README.md

```diff
@@ -151,6 +151,32 @@ Some notes on `make devnet`:
 - feel free to run your own devnet if you are playing around with some custom
   changes to Kakarot.
 
+### Running with [Docker Compose](https://docs.docker.com/compose/)
+
+To orchestrate running a Katana/Madara devnet instance, deploy Kakarot contracts and initialize the RPC, you may use the following commands:
```

Comment:
still no markdownlint in your IDE I guess ! (note: using only vscode from github I review the PR from, I see MD013/line-length: Line length [Expected: 80; Actual: 140]markdownlintMD013
)

Reply:
Managed to fix this and so I also did a fix on other parts of the README